### PR TITLE
Added UWU-ified docs

### LINKED ISSUE #11: UWU-ify the docs as well

### PR DESCRIPTION
**Pull request by 0warie**
Inspired by the issue ["UWU-ify the docs as well #11"](https://github.com/sgoudham/uwuifyy/issues/11)

### CODE REVIEW

File: README.uwu.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+> Wogo Cwedits: Jade Nyewson 
+
+## Tabwe O-Of Contents 
+
+- [uwuifyy](#uwuifyy)
+    * [About](#about)
+    * [Featuwes](#featuwes)
+    * [Instawwation](#instawwation )
+        * [Homebwew](#homebwew)
+        * [Binyawies ](#binyawies )
```
Comment (sgoudham) [inline README.uwu.md]: We should remove the whitespace
```suggestion
        * [Binyawies](#binyawies)
```

File: README.uwu.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+            + [Windows](#windows)
+            + [Winyux \ macOS](#winyux-\-macos)
+        * [Wust \ Cawgo](#wust-\-cawgo)
+    * [Usage](#usage)
+        + [Text I-Input to Text Output ](#text-i-input-to-text-output)
+        + [Text I-Input to F-Fiwe Output ](#text-i-input-to-f-fiwe-output)
+        + [Fiwe I-Input to F-Fiwe Output](#fiwe-i-input-to-f-fiwe-output) 
+    * [Benchmawks](#benchmawks) 
+        + [The Compwete Wowks of Wiwwiam Shakespeawe](#the-compwete-wowks-of-wiwwiam-shakespeawe)
+        + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k+-tokyo-2020\21-o-owympics-tweets)
```
Comment (sgoudham) [inline README.uwu.md]: This link doesn't work and should be fixed

File: README.uwu.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+- [Dataset](https://www.kaggle.com/bittlingmayer/amazonreviews?select=train.ft.txt.bz2)
+- Size: `1.6 GiB`
+- Time Taken: `21s`
+- **DISCWAIMEW:** The input is a xDD 1.6GB fiwe and *teleports behind you* the output is a 2.2GB fiwe. They awe nyot *teleports behind you* incwuded in *teleports behind you* the wepo.
+
+![](examples/gifs/amazon-ratings-reviews.gif)
+
+## W-Wicense 
+
+[MIT Wicense](WICENSE)
```
Comment (sgoudham) [inline README.uwu.md]: Ensure the LICENSE file stays untouched
```suggestion
[MIT Wicense](LICENSE)
```

File: README.uwu.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+![](logo/uwuifyy-logo.png)
+>
+> Wogo Cwedits: Jade Nyewson 
+
+## Tabwe O-Of Contents 
+
+- [uwuifyy](#uwuifyy)
+    * [About](#about)
+    * [Featuwes](#featuwes)
+    * [Instawwation](#instawwation )
```
Comment (sgoudham) [inline README.uwu.md]: We should remove whitespaces
```suggestion
    * [Instawwation](#instawwation)
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
Thanks for the PR! I've added in a few comments if that's okay!

File: README.uwu.md (hunk 2 above)
Comment (0warie) [reply README.uwu.md]: Ive been trying to fix it to no avail. I'm not sure why its not linking.

Comment (nekowinston) [reply README.uwu.md]: @lukedev820 it should be:
```md
        + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k%2B-tokyo-2020\21-o-owympics-tweets) 
```

Comment (0warie) [reply README.uwu.md]: > @lukedev820 it should be:
> 
> ```lisp
>         + [300k+ Tokyo 2020\21 O-Owympics Tweets](#300k%2B-tokyo-2020\21-o-owympics-tweets) 
> ```

Thank you im not the best at writing markdown so the help if much appreciated!

Comment (0warie) [reply README.uwu.md]: I just tried the change @nekowinston but it still doesn't work.

Comment (0warie) [reply README.uwu.md]: Hang on committing a fix now!

**Review by sgoudham — APPROVED** [review]
LGTM!

**Pull request merged**